File: app/src/main/java/cn/chenyuanming/gankmeizhi/application/GankApplication.java
package cn.chenyuanming.gankmeizhi.application;

import android.app.Application;
import android.content.Context;

/**
 * Created by Administrator on 2016/1/25.
 */
public class GankApplication extends Application {
    static GankApplication instance;

    @Override
    public void onCreate() {
        super.onCreate();
        instance = this;
    }

    public static Context getContext() {
        return instance;
    }


}

File: app/src/main/java/cn/chenyuanming/gankmeizhi/application/GankApplication.kt
package cn.chenyuanming.gankmeizhi.application

import android.app.Application
import android.content.Context

/**
 * Created by Chen Yuanming on 2016/1/25.
 */
class GankApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        instance = this
    }

    companion object {
        lateinit var instance: GankApplication

        val context: Context
            get() = instance
    }


}
